Team handles can be used in montior notifcations (#22244)
DataDog/documentation: content/en/monitors/notify/_index.md

{{% notifications-email %}}

#### Teams

> If a [notification channel][17] is set, you can route notifications to a specific Team. Monitor alerts targeting @team-handle will be redirected to the selected communication channel.

Reviewer: Link 17 goes to the Teams documentation, so it's confusing to me to link the "notification channel" text. Consider moving the link to "Team"?
```suggestion
If a notification channel is set, you can route notifications to a specific [Team][17]. Monitor alerts targeting @team-handle are redirected to the selected communication channel.
```

Reviewer 2: [@estherk15 To do in a separate PR] Need to add a note on the Teams page about how to add a notification channel, and then link to those instructions instead of the general Teams page.
